### PR TITLE
Add first subscription metric

### PR DESCRIPTION
**Pull request by gsvr**
- [x] subscriptions.total.sum

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: subscriptions/tasks.py [hunk 1]
```diff
@@ -12,10 +12,33 @@
...
+class FireMetrics(Task):
+
+    """ Fires a metric using the MetricsApiClient
+    """
+
+    def set_metric_client(self):
+        return MetricsApiClient(
+            auth_token=settings.METRICS_AUTH_TOKEN,
+            api_url=settings.METRICS_URL,
+            session=None)
```
Comment (gsvr) [inline subscriptions/tasks.py]: Redundant

File: README.md [hunk 2]
```diff
@@ -1,2 +1,15 @@
...
+  * contentstore
+    * Schedule
+    * MessageSet
+    * BinaryContent
+    * Message
+  * subscriptions
+    * Subscription
+
+## Metrics
+##### subscriptions.total.sum
```
Comment (imsickofmaps) [inline README.md]: I think this is more accurately named `subscription.created.sum` - in aggregate it is the total for a given bucket size, but that is at display time.

File: subscriptions/tasks.py (hunk 1 above)
Comment (imsickofmaps) [inline subscriptions/tasks.py]: Not if you do `def set_metric_client(self, session=None):` and make this `session=session` ... which you should 😄

**Comment by imsickofmaps** [comment]
@gsvr re-reviewed this and thought about all the other metrics we're going to be adding and I think we need to re-look at the logging sender. Because there's no payload checking on `responses` it feels pretty shady relying on the total metrics fired when we really care about the correctness of the data. Thoughts?

File: README.md (hunk 2 above)
Comment (imsickofmaps) [inline README.md]: @gsvr still needs updating

File: subscriptions/tasks.py [hunk 3]
```diff
@@ -12,11 +12,40 @@
...
+
+    """ Fires a metric using the MetricsApiClient
+    """
+
+    def run(self, metric_name, metric_value, session=None, **kwargs):
+        metric_value = float(metric_value)
+        metric = {
+            metric_name: metric_value
+        }
+        try:
```
Comment (imsickofmaps) [inline subscriptions/tasks.py]: Can we take this generic `try` out so we get an exception in Sentry. If we see specific types we want to handle we'll add them in later

**Comment by imsickofmaps** [comment]
Couple of tweaks then :+1: when travis is happy

**Pull request merged**